### PR TITLE
Add digdag server --disable-scheduler option

### PR DESCRIPTION
**Pull request by sonots-zozo**
Implemented the feature request https://github.com/treasure-data/digdag/issues/946.

Recently, I am setting up a staging environment. I want to disable digdag scheduler on staging.

### CODE REVIEW

**Comment by sonots-zozo** [comment]
I could not find any tests for digdag server commands. Please let me know how and where we should write it.

**Comment by sonots-zozo** [comment]
I am glad if this is included in 0.9.36.

File: digdag-core/src/main/java/io/digdag/core/schedule/ScheduleConfig.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package io.digdag.core.schedule;
```
Comment (sonots) [inline digdag-core/src/main/java/io/digdag/core/schedule/ScheduleConfig.java]: I referred AgentConfig.java

File: digdag-core/src/main/java/io/digdag/core/schedule/ScheduleConfigProvider.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package io.digdag.core.schedule;
+
+import java.lang.management.ManagementFactory;
```
Comment (yoyama) [inline digdag-core/src/main/java/io/digdag/core/schedule/ScheduleConfigProvider.java]: Unused import?

Comment (sonots) [reply digdag-core/src/main/java/io/digdag/core/schedule/ScheduleConfigProvider.java]: removed. (also from AgentConfigProvider.java).

**Comment by yoyama** [comment]
LGTM. Thanks!

**Comment by sonots** [comment]
@yoyama @muga I noticed that schedules are shown via `digdag shedules` even if digdag server is working with `--disable-scheduler`. It is mis-leading.

Is it possible to hide or non-register schedules if digdag server starts with `--disable-scheduler`?

**Pull request merged**